workbench-core/swb-common-ui/src/common/utils.ts

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable security/detect-object-injection */
+/* eslint-disable security/detect-unsafe-regex */
 /*
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
 
 export const nameRegex: RegExp = new RegExp('^[A-Za-z]{1}[A-Za-z0-9-\\s]*$');
 
-/* eslint-disable security/detect-unsafe-regex */
```

Comment:
explicitly disabling eslint rules for one line is preferable:
// eslint-disable-next-line security/detect-unsafe-regex